test/unit/annotation_spec.js

```diff
@@ -35,7 +35,6 @@ import {
 import {
   CMAP_URL,
   createIdFactory,
-  getNodeVersion,
```

Comment:
This now seems unused in the project, so let's remove this helper function as well.